templates/debug.php

```diff
@@ -888,3 +931,37 @@ class="dashicons dashicons-download"></i> <?php fs_esc_html_echo_inline( 'Downlo
         });
     </script>
 <?php endif ?>
+<script type="text/javascript">
+    // JavaScript to toggle the visibility of the table body and change the caret icon
+    jQuery( document ).ready( function ( $ ) {
+        $( '.toggle-table' ).each( function () {
+            const button = $( this );
+            const table = button.closest( 'h2' ).next( 'table' );
+            table.css( 'overflow', 'hidden' );
+
+            const toggle = function ( isExpanded ) {
+                button.attr( 'aria-expanded', isExpanded );
+                button.find( '.caret-icon' ).text( isExpanded ? '▼' : '▲' );
+                table.css( {
+                    display  : isExpanded ? 'table' : 'block',
+                    maxHeight: isExpanded ? 'auto' : '0'
+                } );
+            };
+
+            button.on( 'click', function () {
+                toggle( button.attr( 'aria-expanded' ) === 'false' );
+            } );
+
+            toggle( button.attr( 'aria-expanded' ) === 'true' );
+        } );
+    } );
+</script>
+<style>

```

Comment:
@DanieleAlessandra It seems that the styles here should be in the `assets/sass/admin/debug.scss` file.